data_platform/ingestion/sync_twitter.py

```
@@ -122,89 +114,86 @@ def run_sync_tasks(
+def _summarize_twitter_run(metadata: dict[str, Any], output_dir: Path) -> None:
+    print(
+        f"sync_records: wrote {metadata['row_count']} rows to {output_dir} "
+        f"(status={metadata['sync_status']})"
+    )
+
+
+TWITTER_SYNC_SPEC = SyncPlatformSpec(
+    platform="twitter",
+    storage_cls=TwitterStorageManager,
+    entity_label="keywords",
+    init_client=lambda: init_twitter_client(),
```

Ruff (0.16.3)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)